tests/test_utils/www.py

```diff
@@ -29,7 +29,8 @@ def client_with_login(app, expected_response_code=302, **kwargs):
         check_password_hash.return_value = True
         client = app.test_client()
         resp = client.post("/login/", data=kwargs)
-        assert resp.status_code == expected_response_code
+        assert resp.url.raw_path == b"/home"
```

Comment:
```suggestion
        assert resp.url.raw_path == expected_path
```
Maybe we can take `expected_path` as a function parameter.